src/app/new-page/component/content-fallback/content-fallback.component.ts

```diff
@@ -39,9 +36,7 @@ export class ContentFallbackNewComponent implements OnChanges {
               changes['item'].currentValue !== changes['item'].previousValue
             ) {
               this.ready = false;
-              this.content = null;
-              // PIZZA
-              // this.fallback = this.item as KgwContentComplexTypeFallback;
+              this.content = this.item;
```

Comment:
the fallback component is fully handled within the xml parser itself. So once the new parser is live, this component can be removed